400-claims.md

```diff
@@ -50,33 +50,45 @@ The CNAB claim is defined as a JSON document. The specification currently does n
 
 ```json
 {
+  "action": "install",
   "bundle": {
     "credentials": {},
     "images": {},
     "invocationImages": [
       {
-        "image": "technosophos/demo2:0.2.0",
+        "image": "example/helloworld:0.1.0",
         "imageType": "docker"
       }
     ],
-    "name": "technosophos.hellohelm",
-    "outputs": {},
+    "name": "helloworld",
+    "outputs": {
+      "clientCert": {
+        "definition": "x509Certificate",
+        "path": "/cnab/app/outputs/clientCert"
+      },
+      "hostName": {
+        "applyTo": [
+          "install"
+        ],
+        "definition": "string",
+        "description": "the hostname produced installing the bundle",
+        "path": "/cnab/app/outputs/hostname"
+      },
+      "port": {
+        "definition": "port",
+        "path": "/cnab/app/outputs/port"
+      }
+    },
     "parameters": {},
     "schemaVersion": "v1.0.0",
-    "version": "0.1.0"
+    "version": "0.1.2"
   },
-  "bundleReference": "hub.example.com/my/bundle@sha256:eeeeeeeee...",
+  "bundleReference": "hub.example.com/technosophos/hellohelm@sha256:eec03d9da1bad36b3e3a4526cc774153f7024a94f25df8d2dc3ca5602fc5273d",
```

Comment:
should probably remove my username from here, too. I thought I got all of those. Sorry.